### ISSUE TITLE
Mounted directories are empty in the container when using WSL2

### ISSUE THREAD

**Issue by queil** [rancher-sandbox/rancher-desktop#830]
Using Rancher Desktop on Windows + WSL2 integration. When I mount a WSL2 dir it mounts successfully but it's empty in the container even if the mounted dir has files/dirs in it.

**To Reproduce**
1. Enable Wsl2 integration in Rancher Desktop
2. In WSL2: make sure your current working directory has some files/dirs in it
3. In WSL2: run `nerdctl run -it --rm -v $(pwd):/test alpine:latest sh`
2. In the lunched container: run `ls -la /test`

**Result**
No files listed

**Expected behaviour**
I expect to see the files/dirs existing in the WSL2 dir

**Setup (please complete the following information):**
 - OS: Windows 10 (10.0.19043.0) + WSL2 (Ubuntu 20.04.2 LTS)
 - Rancher Desktop version: 0.6.0
 - Kubernetes version: N/A

**Comment by bendem** [comment]
The problem seems to be that the command is executed in the context of the rancher distro, the directory is created there and that's what's mounted inside the container. I did not find a way to mount the directory of the distro you ran the command from.

**Comment by bendem** [comment]
This prevents mounting any directory from WSL into a container. Is there anything we can do to help resolve this? Any pointer to which part of the project would be responsible for this?

**Comment by gunamata** [comment]
It looks like this issue has been resolved with one of the recent Rancher Desktop releases. For example, I tried the steps mentioned in the issue description using the just released [1.4.1](https://github.com/rancher-sandbox/rancher-desktop/releases/tag/v1.4.1) version and found the volume mount working. 

Can you please give this a try with the latest version and let us know if you still see problems?

**Comment by bendem** [comment]
I confirm it's fixed in the latest version (1.4.1).

```
> docker run -it --rm -v /tmp/4259682a-b736-4add-9473-87848a419a6a:/a:ro bash -c 'ls -lh /a'
-rw-r--r--    1 1000     1000      755.6K Jul 19 06:43 /a

> docker run -it --rm -v /tmp/4259682a-b736-4add-9473-87848a419a6a:/a:ro bash -c 'sha256sum /a'
aaadb7d953e06e6ab6629b930b956076496b7064fa4279064affcee6a988ddcb  /a

> sha256sum /tmp/4259682a-b736-4add-9473-87848a419a6a
aaadb7d953e06e6ab6629b930b956076496b7064fa4279064affcee6a988ddcb  /tmp/4259682a-b736-4add-9473-87848a419a6a

**Comment by gunamata** [comment]
I am closing this as this seems to have got resolved in one of the recent releases. 
@queil , @bendem Thank you for confirming.
